src/animation/PropertyBinding.js

```diff
@@ -496,6 +496,25 @@ class PropertyBinding {
 
 					break;
 
+				case 'map':
+
+					if ( ! targetObject.material ) {
+
+						console.error( 'THREE.PropertyBinding: Can not bind to material as node does not have a material.', this );
+						return;
+
+					}
+
+					if ( ! targetObject.material.map ) {
+
+						console.error( 'THREE.PropertyBinding: Can not bind to material.map as node.material does not have a map.', this );
```

Comment:
Is the dot in "material.map" needed? Are we talking about the property "formal" or "common" name?

Reply:
> Is the dot in "material.map" needed? 

Yes.